### ISSUE TITLE
Creating snapshots of stable CM repositories and pulling them 

### ISSUE THREAD

**Issue by gfursin** [mlcommons/ck#952]
Based on the recent feedback from CM users, we should add CM functionality to create a snapshot of CM repositories, upload them to some public or private archive and download them instead of pulling from GitHub, i.e. something like:
```
cm get repo --url=...zenodo.../stable-cm-mlops-20230920.zip
```

It will reduce the size of mlcommons@ck automations from 40MB to < 1MB and will provide stable CM interfaces to ensure reproducibility (we can still ensure it now by --checkout when pulling CM repos from GitHub).

**Comment by arjunsuresh** [comment]
Hi @gfursin is it similar to git checkout of a given commithash/tag without any git history?

**Comment by gfursin** [comment]
> Hi @gfursin is it similar to git checkout of a given commithash/tag without any git history?

Yes, it is similar. The difference is that we can add snapshots of CM repos to permanent archives (or private drives) and download them as zip/tar ...

**Comment by arjunsuresh** [comment]
Thank you Grigori. Got it. I was thinking for generic git repos and not just CM repos.

**Comment by gfursin** [comment]
Done.
